src/BitBucketOnPrem.ts

```diff
@@ -30,52 +39,102 @@ export interface BitBucketOnPremInput {
 const fetchNoCache = (requestInfo: RequestInfo, requestInit: RequestInit) => {
     requestInit = requestInit || {};
     if (!requestInit?.cache) {
-        requestInit.cache = 'no-store'
+        requestInit.cache = 'no-store';
     }
     if (!requestInit?.credentials) {
         requestInit.credentials = 'omit';
     }
-    return fetch(requestInfo.toString().replace('scm/', '').replace('scm%2F',''), requestInit);
-}
+    return fetch(requestInfo.toString().replace('scm/', '').replace('scm%2F', ''), requestInit);
+};
+
+export const getFileTreeByPath =
+    async ({url, accessToken, projectKey, repoName, commit, filePath}: BitbucketOnPrem): Promise<string[]> => {
+        const templateUrl: string =  addSlugToUrl(TREE_FETCH_URL.BY_PATH, filePath);
```

Comment:
double space before addSlugToUrl.
Also, why is the TREE_FETCH_URL const needed?